tests/test_tester.py

```diff
@@ -10,9 +10,9 @@ def test_tester_basic():
     tester = fault.Tester(circ)
```

Comment:
the serialization tests should be disjoint from the tester tests. the tester tests should just ensure that the series of actions generated is accurate (mostly trivial).

Reply:
Done,